Dockerfile-slim.template

```diff
@@ -27,13 +32,16 @@ RUN set -ex \
        libspatialite-dev \
        libpq-dev \
        libcurl4-gnutls-dev \
-       libproj-dev \
        libxml2-dev \
        libgeos-dev \
        libnetcdf-dev \
        libpoppler-dev \
        libhdf4-alt-dev \
        libhdf5-serial-dev \
+       libpoppler-private-dev \
+       sqlite3 \
+       libsqlite3-dev \
+       libtiff-dev \
```

Comment:
Are these dependencies required for compiling `libproj`?

Reply:
@rbreslow yes, proj v7 depends on sqlite and libtiff now. 
Starting v6 it depends on sqlite and starting v7 it requires libtiff and curl.